agent-core/src/main/java/com/newrelic/agent/android/Agent.java

```diff
@@ -212,5 +213,11 @@ public static boolean isInstantApp() {
         return getImpl().isInstantApp();
     }
 
+    public static void persistHarvestDataToDisk(String data){
+        getImpl().persistHarvestDataToDisk(data);
+    }
```

Comment:
Double check: What would happen if data is null or empty? Or would data ever be null or empty?